staking_deposit/cli/generate_bls_to_execution_change.py

```diff
@@ -0,0 +1,195 @@
+import os
+import click
+import json
+from typing import (
+    Any,
+    Sequence,
+)
+
+from eth_typing import HexAddress
+
+from staking_deposit.credentials import (
+    CredentialList,
+)
+from staking_deposit.utils.validation import (
+    validate_bls_withdrawal_credentials_list,
+    validate_bls_withdrawal_credentials_matching,
+    validate_eth1_withdrawal_address,
+    validate_int_range,
+    verify_bls_to_execution_change_json,
+    validate_validator_indices,
+)
+from staking_deposit.utils.constants import (
+    DEFAULT_BLS_TO_EXECUTION_CHANGES_FOLDER_NAME,
+    MAX_DEPOSIT_AMOUNT,
+)
+from staking_deposit.utils.click import (
+    captive_prompt_callback,
+    choice_prompt_func,
+    jit_option,
+)
+from staking_deposit.exceptions import ValidationError
+from staking_deposit.utils.intl import (
+    closest_match,
+    load_text,
+)
+from staking_deposit.settings import (
+    ALL_CHAINS,
+    MAINNET,
+    PRATER,
+    get_chain_setting,
+    get_devnet_chain_setting,
+)
+from .existing_mnemonic import (
+    load_mnemonic_arguments_decorator,
+)
+
+
+def get_password(text: str) -> str:
+    return click.prompt(text, hide_input=True, show_default=False, type=str)
+
+
+FUNC_NAME = 'generate_bls_to_execution_change'
+
+
+@click.command()
+@jit_option(
+    default=os.getcwd(),
+    help=lambda: load_text(['arg_bls_to_execution_changes_folder', 'help'], func=FUNC_NAME),
+    param_decls='--bls_to_execution_changes_folder',
+    type=click.Path(exists=True, file_okay=False, dir_okay=True),
+)
+@jit_option(
+    callback=captive_prompt_callback(
+        lambda x: closest_match(x, list(ALL_CHAINS.keys())),
+        choice_prompt_func(
+            lambda: load_text(['arg_chain', 'prompt'], func=FUNC_NAME),
+            list(ALL_CHAINS.keys())
+        ),
+    ),
+    default=MAINNET,
+    help=lambda: load_text(['arg_chain', 'help'], func=FUNC_NAME),
+    param_decls='--chain',
+    prompt=choice_prompt_func(
+        lambda: load_text(['arg_chain', 'prompt'], func=FUNC_NAME),
+        # Since `prater` is alias of `goerli`, do not show `prater` in the prompt message.
+        list(key for key in ALL_CHAINS.keys() if key != PRATER)
+    ),
+)
+@load_mnemonic_arguments_decorator
```

Comment:
can we have a prompt or something to make sure the user is doing this offline? to get the signatures in an airgapped location or something.